packages/flutter/test/material/data_table_test.dart

```diff
@@ -46,6 +46,9 @@ void main() {
             onSelectChanged: (bool? selected) {
               log.add('row-selected: ${dessert.name}');
             },
+            onHover: (bool? selected) {
```

Comment:
This test is not quite sufficient. This sets the property, but it does not verify that the callback is triggered by a hover event.